models/engine/db_storage.py

```diff
@@ -102,5 +102,5 @@ def get(self, cls, id):
         '''
         try:
             return [a for a in self.all(cls).values() if a.id == id][0]
```

Comment:
Your variable name is shallow and unscholarly . I implore you to make due adjustments.